charts/outline/templates/secret.yaml

```diff
@@ -6,7 +6,8 @@ metadata:
     {{- include "outline.labels" . | nindent 4 }}
 data:
   AWS_SECRET_ACCESS_KEY: {{ .Values.outline.aws.secret | b64enc | quote }}
-  DATABASE_URL: {{ include "outline.postgresqlUrl" . | b64enc | quote }}
+  DATABASE_URL: {{ .Values.outline.postgresqlUrl | b64enc | quote }}
```

Comment:
Not sure I'm getting why you want to do this, but you'll probably want to drop the related helper here (https://github.com/MnrGreg/helm-charts/blob/master/charts/outline/templates/_helpers.tpl#L65-L74)

Reply:
reverted the helper and removed the quotes as it was throwing a parsing error

```shell
yarn run v1.22.5
$ sequelize db:migrate --env=production-ssl-disabled

Sequelize CLI [Node: 14.16.1, CLI: 6.2.0, ORM: 6.3.5]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".

ERROR: Error parsing url: "postgres://pgadmin:1ejq-redacted-H5O8@pg-outlinewiki:5432/outline"
```